gradle.properties

```diff
@@ -3,4 +3,4 @@ org.gradle.configureondemand=false
 org.gradle.jvmargs=-XX:MaxMetaspaceSize=768m
 localrepo=build/mockk-repo
 # localrepo=/Users/raibaz/.m2/repository
-# kotlin.version=1.5.10
+kotlin.version=1.5.31
```

Comment:
I think this should stay commented (or even better removed), because the kotlin version is being set externally by the github action that runs the builds for several different versions.

It's likely the reason why builds on 1.3.* fail.